ledger/ledgercore/statedelta.go

```diff
@@ -78,6 +90,11 @@ type StateDelta struct {
 	// previous block timestamp
 	PrevTimestamp int64
 
+	// Modified local creatable states. The value is true if the creatable local state
+	// is created and false if deleted. Used by indexer.
+	ModifiedAssetHoldings  map[AccountAsset]bool
+	ModifiedAppLocalStates map[AccountApp]bool
```

Comment:
since these two share an index, I would suggest making it into a map[CreatableIndex]CreatableType which would have a very similar cost, but would pack it all in one.